scripts/transactions/getWinner.ts

```diff
@@ -19,7 +19,7 @@ async function getwinningProposal(contractAddress: string): Promise<void> {
   } catch (err) {
     throw new Error(err as string);
   }
-  console.log(`The winner is ${winningProposalName} with ${winningProposalVoteCount} votes`);
+  console.log(`The winner is ${winningProposalName} at index: ${winningProposalIndex}`);
```

Comment:
You can just nest the try/catch blocks instead of reassigning the variables

```
try {
    const winningProposalIndex = await ballot.winningProposal();
    try {
        const winningProposalNameBytes32 = await ballot.winnerName();
        const winningProposalName = ethers.utils.parseBytes32String(winningProposalNameBytes32);
        console.log(`The winner is ${winningProposalName} at index: ${winningProposalIndex}`);
      } catch (err) {
        throw new Error(err as string);
      }
} catch (err) {
   throw new Error(err as string);
}

Reply:
That's better, thanks.

Reply:
done.